subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/component_metadata_rules.adoc

```diff
@@ -38,6 +38,47 @@ A service which is helpful for advanced usage of component metadata rules with c
 A component metadata rule can be applied to all modules — `all(rule)` — or to a selected module — `withModule(groupAndName, rule)`.
 Usually, a rule is specifically written to enrich metadata of one specific module and hence the `withModule` API should be preferred.
 
+[[sec:rules_in_settings]]
+=== Declaring rules in a central place
+
+NOTE: Declaring component metadata rules in settings is an incubating feature
+
+Instead of declaring rules on a project's component metadata handler, it is possible to declare rules in the `settings.gradle(.kts)` file.
```

Comment:
```suggestion
Instead of declaring rules for each subproject individually, it is possible to declare rules in the `settings.gradle(.kts)` file for the whole build.
```

To clarify "single project" vs "whole build" and because the term 'component metadata handler' is used nowhere else in this documentation section.